consensus/ibft/src/integration-test/java/tech/pegasys/pantheon/consensus/ibft/support/MessageReceptionHelpers.java

```diff
@@ -52,15 +53,36 @@ public static void assertPeersReceivedExactly(
             messageMatchesExpected(rxMsgData, msg);
           });
     }
-    allPeers.forEach(p -> p.clearReceivedMessages());
+    allPeers.forEach(ValidatorPeer::clearReceivedMessages);
+  }
+
+  @SafeVarargs
+  public static void assertPeersContainsReceivedMessages(
```

Comment:
maybe rename?
assertMessagesReceivedByPeersContains, or
assertPeersReceivedAtLeast?

Reply:
removed this method as part of refactoring to use existing assert messages method